cmd/ctr/shim.go

```diff
@@ -214,10 +216,18 @@ var shimExecCommand = cli.Command{
 		if err != nil {
 			return err
 		}
+
+		// read spec file and extract Any object
+		spec, err := ioutil.ReadFile(context.String("spec"))
```

Comment:
We will change this in a future PR when we get the image config to spec conversion finished.